### PR TITLE
Alternative pages updates

### PR DESCRIPTION
**Pull request by bleonard**
## Change description

Updating based on changes

### Development

- [x] Application changes have been tested appropriately

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [ ] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: pages/solutions/segment-alternative.mdx [hunk 1]
```diff
@@ -78,7 +78,7 @@ Grouparoo’s fundamental model is built upon querying source-of-truth data that
   heading="Easily define records, no matter your data."
   width="713"
   height="387"
-  content="Grouparoo's queryable data structure makes it easy to create records -- even from historical data.  You can configure customer records from the command line on our community edition, or straight from the UI with our enterprise product."
+  content="Grouparoo's queryable data structure makes it easy to create records -- even from historical data.  You can configure customer records from the developer UI on our community edition, or straight from the app UI with our enterprise product."
```
Comment (evantahler) [inline pages/solutions/segment-alternative.mdx:81]: I think it was right before - the developer UI doesn't let you do things in the community edition... or perhaps you are including config in the "edition"?

Comment (evantahler) [reply pages/solutions/segment-alternative.mdx:81]: Yeah - I think that's what you are saying thoughout.

File: pages/solutions/hightouch-alternative.mdx [hunk 2]
```diff
@@ -17,29 +11,34 @@ comparisonChartData:
...
-          comp: "Manual, hourly, daily, or weekly",
+          feature: "Multiple Sources for a Data Model",
+          grouparoo: true,
+          comp: false,
         },
         {
-          feature: "Configurable via UI",
-          grouparoo: [true, "(enterprise edition)"],
-          comp: true,
+          feature: "Option to mode data without SQL (no-code)",
```
Comment (evantahler) [inline pages/solutions/hightouch-alternative.mdx]: Option to model? Optional mode?

File: pages/solutions/census-alternative.mdx [hunk 3]
```diff
@@ -11,54 +11,50 @@ comparisonChartData:
...
         {
-          feature: "Easy to configure private vs. shareable data",
+          feature: "Multiple Sources for a Data Model",
           grouparoo: true,
           comp: false,
         },
         {
-          feature: "Configurable via UI",
-          grouparoo: [true, "(enterprise edition)"],
+          feature: "Option to mode data without SQL (no-code)",
```
Comment (evantahler) [inline pages/solutions/census-alternative.mdx]: (same)

**Review by evantahler — APPROVED** [review]
some minor grammar stuff, but 👍 othewise

File: pages/solutions/segment-alternative.mdx (hunk 1 above)
Comment (bleonard) [reply pages/solutions/segment-alternative.mdx:81]: Yeps!

**Pull request merged**